rust/Makefile

```diff
@@ -150,3 +150,7 @@ $(objtree)/rust/kernel.o: private rustc_target_flags = --extern alloc \
 $(objtree)/rust/kernel.o: $(srctree)/rust/kernel/lib.rs $(objtree)/rust/alloc.o \
     $(objtree)/rust/libmodule.so $(objtree)/rust/bindings_generated.rs FORCE
 	$(call if_changed_dep,rustc_library)
+
+GENERATE_RUST_ANALYZER = $(srctree)/scripts/generate_rust_analyzer.sh
+rust-analyzer:
+	$(Q)$(GENERATE_RUST_ANALYZER) $(srctree) $(objtree) $(RUST_LIB_SRC) $(abspath $(objtree)/rust/bindings_generated.rs) > $(objtree)/rust-project.json
```

Comment:
Why `abspath`? Does your script rely on that?

Also, it looks like there is no newline at the end (i.e. the "stop" symbol).

Reply:
As far as I understand rust-analyser this abspath is needed. Newline I forgot, I can add one after my exam, tomorrow. (shut down my dev setup for today :-) )

Reply:
Good luck on it! :-)